README.md

```diff
@@ -114,6 +125,38 @@ catcher3.disable()
 bp = d.breakpoint(0xdeadc0de, hardware=True)
 
 d.cont()
+d.wait()
+
+d.gdb()
+```
+
+## ASAP Polling (Pwning mode)
+libdebug also allows you to make all commands execute as soon as possible, without having to wait for a stopping event. To enable this mode, you can use the `auto_interrupt_on_command=True` 
+
+```python
+from libdebug import debugger
+
+d = debugger("/path/to/executable", auto_interrupt_on_command=True)
+
+pipes = d.run()
+
+bp = d.breakpoint("function")
+
+d.cont()
+
+# Read shortly after the cont is issued
+# The proess is forcibly stopped to read the register
```

Comment:
process*

Reply:
Addressed in [9e4a7f8](https://github.com/libdebug/libdebug/pull/129/commits/9e4a7f831a709f7ca168593df7deb81678b86ed5)